hack/check-everything.sh

```diff
@@ -19,7 +19,7 @@ set -e
 hack_dir=$(dirname ${BASH_SOURCE})
 source ${hack_dir}/common.sh
 
-k8s_version=1.14.1
+k8s_version=1.16.4
```

Comment:
Can we mention this version upgrade in the PR message itself. Is CRD with version v1 supported from 1.16.4?

Reply:
Added a message